src/pages/index.vue

```diff
@@ -1,16 +1,26 @@
 <script setup lang="ts">
-useHead({
-  title: 'Home Page',
-})
+useHead({ title: 'Home Page' })
+const { t } = useI18n()
 </script>
 
 <template>
   <div>
     <RouterLink to="/test" data-testid="test-link">
-      Test
+      {{ t('test') }}
     </RouterLink>
```

Comment:
Like above, I think it would be better to keep only one of the `Home` or `test` keywords for this components